packages/web5/src/web5.ts

```diff
@@ -68,6 +70,7 @@ export class Web5 {
   private constructor(options: Web5Options) {
     this.#connectedDid = options.connectedDid;
     this.dwn = new DwnApi(options.web5Agent, this.#connectedDid);
+    this.vc = new VcApi(options.web5Agent, this.#connectedDid);
```

Comment:
this.#connectedDid gives the private key information right? 
If I read this correct it gives the JWK and the algo